superx/information_extractors/item_info_extractor.py

```diff
@@ -1,12 +1,15 @@
+'''
+imports
+'''
 from datetime import datetime
 import gzip
 import logging
 import xml.etree.ElementTree as ET
 from decimal import Decimal
-from bs4 import BeautifulSoup
-import requests
-from models import Product, BranchPrice
-from app import supermarket_info_dictionary, session, db
+from bs4 import BeautifulSoup # pylint: disable=import-error
+import requests # pylint: disable=import-error
+from models import Product, BranchPrice # pylint: disable=import-error
+from app import supermarket_info_dictionary, session # pylint: disable=import-error
```

Comment:
same thing here. see my pr

Reply:
not sure if its a good idea to both have the same changes in our PRs i think one one of us has to rebase no?

Reply:
if not i can change no problem

Reply:
I have no idea what rebase means, but i think sim should accept ur pr and ill try and change mine

Reply:
ok i changed to yours